### PR TITLE
Feature/cash 224

### PR DESCRIPTION
**Pull request by dubdabasoduba**
+ add some javadocs in the api files
+ addition of the javadocs and changes on some of the magic numbers edits
+ complete API checkstyle and reformatting
+ complete check-style in the omod_1.x	
+ Complete the check-style omod_2.x"

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -110,6 +110,33 @@
 				<filtering>true</filtering>
 			</testResource>
 		</testResources>
+
+		<plugins>
+			<plugin>
+				<groupId>com.googlecode.maven-java-formatter-plugin</groupId>
```
Comment (k-joseph) [inline api/pom.xml]: what is this plugin used for?

Comment (dubdabasoduba) [inline api/pom.xml]: It is to help in the formatting of code using a standard format style acceptable by the whole team. Just to make our code uniform

Comment (k-joseph) [inline api/pom.xml]: I thought this was already happening! I think there's more to define if that's the case, see one at https://github.com/openmrs/openmrs-core/blob/master/pom.xml

**Comment by k-joseph** [comment]
Sounds to be a good idea to always separate changes added from the formatted code to make easy the code review experience

File: api/pom.xml (hunk 1 above)
Comment (akmad) [inline api/pom.xml]: @k-joseph This is just the api/pom.xml. The top-level pom file has the same changes as the pom file you linked.

Comment (k-joseph) [inline api/pom.xml]: sure, i was that after adding this comment and  i forgot removing it

**Pull request merged**